src/WebView.js

```diff
@@ -32,6 +32,11 @@ class WebView extends WebKit.WebView {
     this.notify("is-online");
   }
 
+  load_uri(uri) {
+    this.#updateIsOnline(uri);
+    super.load_uri(uri);
+  }
+
```

Comment:
Do we need this? If ``load_uri`` is called then ``notify::uri`` would always get triggered right?